docs/bundle.js.LICENSE.txt

```diff
@@ -129,3 +138,7 @@ object-assign
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
  * SOFTWARE.
  */
+
+//! moment.js
+
+//! moment.js locale configuration
```

Comment:
What's with all this license updates? 😮

Reply:
I feel like I usually get this whenever I run `make docs`